### ISSUE TITLE
Incorrect ListView subitem ControlType in edit mode

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#4032]
* .NET Core Version: 
.Net SDK 5.0.100-rc.2.20477.2

* Have you experienced this same bug with .NET Framework?: 
No

* More Info
This is a regression issue, cannot repro in .Net Core 3.1

**Minimal repro:**
1. Open the attached WinFormsApp9.zip application in VS, then build and run it.
[WinFormsApp9.zip](https://github.com/dotnet/winforms/files/5302987/WinFormsApp9.zip)
2. Launch Inspect to focus on ListView item text in edit state

**Actual behavior:**
In Inspect, the listView items text’s ControlType, LocalizedControlType, LegacyIAccessible.Role properties are not correct when listView LabelEdit is true
![image](https://user-images.githubusercontent.com/26474449/94639828-af936f80-030f-11eb-8b1c-2e9095b119fc.png)

**Expected behavior:**
In Inspect, the listView items text’s ControlType, LocalizedControlType, LegacyIAccessible.Role properties are correct when listView LabelEdit is true
![image](https://user-images.githubusercontent.com/26474449/94639852-c20da900-030f-11eb-805c-56024a8d72a1.png)

**Comment by Zheng-Li01** [comment]
Verified the issue with latest NET SDK 6.0.100-alpha.1.20557.2, the issue has been fixed that have correct ListView subitem ControlType in edit mode. see below screenshot.
![UpdateCoreIssue](https://user-images.githubusercontent.com/38325459/98524648-6e995e00-22b2-11eb-993f-1cc74f053c8f.PNG)
